src/catcollar/mod.rs

```diff
@@ -500,8 +499,8 @@ fn pack_result(rt: &IoUringRuntime, result: OperationResult, qd: QDesc, qt: u64)
         OperationResult::Pop(addr, bytes) => match rt.into_sgarray(bytes) {
             Ok(mut sga) => {
                 if let Some(endpoint) = addr {
-                    sga.sga_addr.sin_port = endpoint.get_port().into();
-                    sga.sga_addr.sin_addr.s_addr = u32::from_le_bytes(endpoint.get_address().octets());
+                    sga.sga_addr.sin_port = endpoint.port().into();
+                    sga.sga_addr.sin_addr.s_addr = u32::from_le_bytes(endpoint.ip().octets());
```

Comment:
I understand you didn't write the original version of this, but this seems awfully convoluted just to (essentially) assign a u32 to another u32.  Couldn't the right-hand side of this just be endpoint.ip()?

Even more generally, couldn't both of these lines be replaced now by "sga.sga_addr = endpoint"?  They're both really just sockaddr_in under the covers.

Reply:
Solved the `into()` call.

Reply:
I think we decided to leave this for now, or at least a different PR.